src/Utils/Embed.php

```diff
@@ -74,18 +99,6 @@ private function cleanIframe(?string $html): ?string
         }
 
         return $html;
-
-        //        $types = [
```

Comment:
can this block be cleaned up? not sure if it was used for debugging purposes

Reply:
these lines are removed already right? It's in red.

Reply:
🤦